packages/editor/src/lib/options.ts

```diff
@@ -67,10 +66,20 @@ export interface TldrawOptions {
 	 */
 	readonly exportProvider: ComponentType<{ children: React.ReactNode }>
 	/**
-	 * How should the note shape resize? By default it does not resize (except automatically based on its text content),
-	 * but you can set it to be user-resizable using scale.
+	 * Options that relate to tldraw's built-in default shapes.
 	 */
-	readonly noteShapeResizeMode: 'none' | 'scale'
+	readonly shapes: {
+		draw: {
+			maxPoints: number
+		}
+		note: {
+			/**
+			 * How should the note shape resize? By default it does not resize (except automatically based on its text content),
+			 * but you can set it to be user-resizable using scale.
+			 */
+			resizeMode: 'none' | 'scale'
+		}
+	}
```

Comment:
the docs for this won't show up properly. For  the doc comments to show, you need to use a separate interface for each of these objects